pkg/router/template/router.go

```diff
@@ -438,6 +439,9 @@ func (r *templateRouter) watchMutualTLSCert() error {
 			log.V(0).Info("failed to establish watch on mTLS certificate directory", "error", err)
 			return nil
 		}
+		if err := crl.UpdateCRLFile(caPath, "/crls-test"); err != nil {
```

Comment:
Why does the path end in "-test"? Is this a temporary location?

Reply:
Yes, it was just a temp location. The CRLs are now being stored in temp dirs in `/var/lib/haproxy/mtls`.